### PR TITLE
setting up webpack

### LINKED ISSUE #1: initialize project with webpack
We need to initialize our react project with webpack.

### PR DESCRIPTION
**Pull request by nj-io**
closes #1
wrote basic .babelrc, .eslintrc.json, webpack.config.js files and installed init modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.4.0",
+    "eslint-plugin-react": "^7.5.1",
+    "prettier": "^1.10.2",
+    "webpack-dev-server": "^2.10.1"
+  },
+  "scripts": {
+    "lint": "eslint **/*.{js,jsx} --quiet",
```
Comment (claireinez) [inline package.json]: A `start` script would be handy.

**Review by claireinez — COMMENTED** [review]
It would be useful to have a `start` script and also basic running instructions in the README.

**Comment by claireinez** [comment]
@ionush could you please update the comment for this PR to "Closes #1"? That way, GitHub will automatically close the relevant issue once it is merged into master.

**Comment by Karyum** [comment]
@ionush https://github.com/FACN3/cryptoCoin/pull/2#issuecomment-357509281

**Review by claireinez — APPROVED** [review]

**Pull request merged**